pkg/udf/map_udf.go

```diff
@@ -156,13 +160,40 @@ func (u *MapUDFProcessor) Start(ctx context.Context) error {
 			}
 		}()
 
+	} else if enableBatchMapUdf {
+		// if Batch Map mode is enabled create the client and handler for that accordingly
+
+		// Wait for server info to be ready
+		serverInfo, err := sdkserverinfo.SDKServerInfo(sdkserverinfo.WithServerInfoFilePath(sdkclient.BatchMapServerInfoFile))
+		if err != nil {
+			return err
+		}
+
+		// create the client and handler for batch map interface
+		batchMapClient, err := batchmapper.New(serverInfo, sdkclient.WithMaxMessageSize(maxMessageSize))
+		if err != nil {
+			return fmt.Errorf("failed to create batch map client, %w", err)
+		}
+		batchMapHandler = rpc.NewUDSgRPCBasedBatchMap(vertexName, batchMapClient)
+		// Readiness check
+		if err := batchMapHandler.WaitUntilReady(ctx); err != nil {
+			return fmt.Errorf("failed on batch map UDF readiness check, %w", err)
+		}
+		defer func() {
+			err = batchMapHandler.CloseConn(ctx)
+			if err != nil {
+				log.Warnw("Failed to close gRPC client conn", zap.Error(err))
+			}
+		}()
 	} else {
+		// As a vertex can be either Map mode or Batch mode we reuse the same server-info file and socket file for both
```

Comment:
Is the comment correct?

Reply:
It was an older comment when vertex was unified. Updated